extensions/objects/templates/positive-int-editor.directive.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2012 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for positive int editor.
+ */
+
+// Every editor directive should implement an alwaysEditable option. There
+// may be additional customization options for the editor that should be passed
+// in via initArgs.
+
+angular.module('oppia').directive('positiveIntEditor', [
```

Comment:
Would it be possible to write this file in angular? /cc @srijanreddy98... not sure

Reply:
Yep, I can do this!

Reply:
Done -- converted to angular